src/NuGet.Clients/NuGet.PackageManagement.PowerShellCmdlets/PackageActionBaseCommand.cs

```diff
@@ -167,7 +167,7 @@ protected async Task InstallPackageByIdentityAsync(NuGetProject project, Package
                 }
                 else
                 {
-                    throw ex;
+                    throw;
```

Comment:
I think it would be cleaner to use a `when` clause on line 159 instead, what do you think?

```c#
catch (InvalidOperationException ex) when (ex.InnerException is PackageAlreadyInstalledException)
{
}
```

That would get rid of this entire `if...else`

Reply:
Thanks! I agree. Applied~